UNRELEASED.md

```diff
@@ -1,5 +1,5 @@
-`@skyscanner/backpack-web`
+`bpk-mixins`: `39.0.1=> 39.1.0`
+`bpk-stylesheets`: `9.0.1 => 9.0.2`
 
 **Changed:**
-  - `BpkDividedCard`
-    - Add prop `isElevated` to switch on/off the card shadow so that it wouldn't overlap with the shadow of the card wrapper.
```

Comment:
This change is still needed as it hasn't been released yet so will require being added back

Reply:
Sure thing, will add them back.